test/fw/bin/pbs_benchpress

```diff
@@ -340,6 +348,42 @@ if __name__ == '__main__':
             'Testsuites names require (see -t) along with -i option!')
         sys.exit(1)
 
+    try:
+        tc_failure_threshold = int(tc_failure_threshold)
+        if tc_failure_threshold < 0:
+            raise ValueError
+    except:
```

Comment:
@arungrover I have updated design document to add additional information about owner and permission of directory in which post analysis data will be stored.